### PR TITLE
Blacklist xvs claimers (backport from master)

### PR DESCRIPTION
**Pull request by kkirka**
This PR brings the changes introduced in https://github.com/VenusProtocol/venus-protocol/pull/156 from master to develop so that we could merge develop to master without getting any conflicts

### CODE REVIEW

**Review by coreyar — APPROVED** [review]

File: contracts/Comptroller.sol [hunk 1]
```diff
@@ -1272,17 +1264,23 @@ contract Comptroller is ComptrollerV7Storage, ComptrollerInterfaceG2, Comptrolle
...
             }
             if (suppliers) {
                 updateVenusSupplyIndex(address(vToken));
                 for (j = 0; j < holders.length; j++) {
                     distributeSupplierVenus(address(vToken), holders[j]);
-                    venusAccrued[holders[j]] = grantXVSInternal(holders[j], venusAccrued[holders[j]], shortfalls[j], collateral);
                 }
             }
         }
+
```
Comment (0xlucian) [inline contracts/Comptroller.sol:1276]: Just to make sure I understand the logic... In the upper loops we destribute XVS for the holders for each vToken from the list. And in the last loop, we just grant the distributed so far rewards? @kkirka

File: contracts/Comptroller.sol [hunk 2]
```diff
@@ -1272,17 +1264,23 @@ contract Comptroller is ComptrollerV7Storage, ComptrollerInterfaceG2, Comptrolle
...
+        for (j = 0; j < holders.length; j++) {
```
Comment (0xlucian) [inline contracts/Comptroller.sol:1277]: Just to make sure I understand the logic... In the upper loops we destribute XVS for the holders for each vToken from the list. And in the last loop, we just grant the distributed so far rewards? @kkirka

Comment (kkirka) [reply contracts/Comptroller.sol:1277]: Yep, `grantXVSInternal` does the actual transfer, while `distribute*` just increases venusAccrued but doesn't transfer tokens

**Review by 0xlucian — APPROVED** [review]
LGTM

**Pull request merged**